docker-compose.yml

```diff
@@ -93,6 +93,14 @@ services:
             ca.mcgill.a11y.image.port: 5000
         env_file:
             - ./config/azure-api.env
+    openstreet:
+        build:
+            context: ./preprocessors/openstreet
+            dockerfile: Dockerfile
+        image: "osm-preprocessors:latest"
```

Comment:
Change this to the ghcr one for production. "osm-preprocessors:latest" should only be used for your local testing on unicorn